aws_lambda_builders/workflows/python_pip/packager.py

```diff
@@ -192,6 +192,9 @@ class DependencyBuilder(object):
         "manylinux1_x86_64": "manylinux_2_5_x86_64",
         "manylinux2010_x86_64": "manylinux_2_12_x86_64",
         "manylinux2014_x86_64": "manylinux_2_17_x86_64",
+        "manylinux1_aarch64": "manylinux_2_5_aarch64",
+        "manylinux2010_aarch64": "manylinux_2_12_aarch64",
+        "manylinux2014_aarch64": "manylinux_2_17_aarch64",
```

Comment:
Where did you get these mapping from?

Reply:
I was using this PR to test if my assumption is correct or not, and I marked it as draft, but seems I made it ready by mistake .. sorry for that .. I still need to work on it, it is not finalized